subsys/net/ip/net_context.c

```diff
@@ -122,6 +122,13 @@ static uint16_t find_available_port(struct net_context *context,
 #define find_available_port(...) 0
 #endif
 
+bool net_context_port_in_use(enum net_ip_protocol ip_proto,
+			   uint16_t local_port,
+			   const struct sockaddr *local_addr)
+{
+	return check_used_port(ip_proto, local_port, local_addr) != 0;
```

Comment:
As you noticed, the local_port given to `check_used_port()` should be in network byte order. Easy fix is just to have `htons(local_port)` here. Lets leave host byte order in `net_context_port_in_use()`, it is more clearer that way.

Reply:
Ah, I see. Will do